java/arcs/core/data/proto/manifest.proto

```diff
@@ -197,3 +201,66 @@ message UnaryExpressionProto {
   RefinementExpressionProto expr = 1;
   OPERATOR operator = 2;
 }
+
+message AccessPathProto {
+  message Selector {
+    oneof selector {
+      string field = 1;
+      // TODO(bgogul): Other selectors like dereferencing, index, etc.
+    }
+  }
+  string handle_connection = 1;
+  repeated Selector selectors = 2;
+}
+
+message InformationFlowLabelProto {
+  message Predicate {
+    message Literal {
+      InformationFlowLabelProto label = 1;
+    }
```

Comment:
I wonder whether this one is necessary? You could just replace where it's used in the oneof below with a label directly.

Reply:
ah, good idea. Done.